server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/draft/JmapGuiceProbe.java

```diff
@@ -103,4 +106,12 @@ public State getLastestState(AccountId accountId) {
     public State getLastestStateWithDelegation(AccountId accountId) {
```

Comment:
should we refactor those kind of method name, as we have the state on other objects than Mailboxes now?